test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestMisc.java

```diff
@@ -39,7 +39,7 @@
 public class TestMisc {
 
 	private static Logger logger = Logger.getLogger(TestMisc.class);
-	public static final boolean isJava9 = "9".equalsIgnoreCase(System.getProperty("java.specification.version"));
+	public static final boolean isJava9AndUp = !"1.8".equals(System.getProperty("java.specification.version"));
```

Comment:
Can this be changed to use `VersionCheck.isMajor()`?  https://github.com/eclipse/openj9/blob/f66506e7a528340d176260c3af47c63cf66979ec/test/functional/TestUtilities/src/org/openj9/test/util/VersionCheck.java

That way we only have 1 place that determines the current version